assets/kube-burner-report-ocp-wrapper/queries.libsonnet

```diff
@@ -875,18 +875,18 @@ local elasticsearch = g.query.elasticsearch;
           + elasticsearch.bucketAggs.Terms.settings.withOrder('desc')
           + elasticsearch.bucketAggs.Terms.settings.withOrderBy('_term')
           + elasticsearch.bucketAggs.Terms.settings.withMinDocCount(0)
-          + elasticsearch.bucketAggs.Terms.settings.withSize("10"),
+          + elasticsearch.bucketAggs.Terms.settings.withSize("0"),
           elasticsearch.bucketAggs.Terms.withField("labels.resource.keyword")
           + elasticsearch.bucketAggs.Terms.withId("3")
           + elasticsearch.bucketAggs.Terms.withType('terms')
           + elasticsearch.bucketAggs.Terms.settings.withOrder('desc')
           + elasticsearch.bucketAggs.Terms.settings.withOrderBy('_term')
           + elasticsearch.bucketAggs.Terms.settings.withMinDocCount('1')
-          + elasticsearch.bucketAggs.Terms.settings.withSize("10"),
+          + elasticsearch.bucketAggs.Terms.settings.withSize("0"),
```

Comment:
Does this look alright fetching all resource types? I recall there being a large number that don't look particularly nice if we remove the limit on resources.

Reply:
I did change it for the panels that were not meant to view top k/n results. But I am open to other suggestions too.

Reply:
@afcollins just a gentle reminder on this PR.

Reply:
Took a look. You fixed one of my gripes with this change. :)